### PR TITLE
Create initial documentation

### PR DESCRIPTION
**Pull request by sethmlarson**
After merge I'll add the `docs/readthedocs.org:truststore` GitHub commit status to be required.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Truststore
+==========
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+Verify certificates using OS trust stores. Supports macOS, Windows, and Linux (with OpenSSL).
+
```
Comment (davisagli) [inline docs/source/index.rst:9]: Should we specify our Python version support here too?

**Review by davisagli — COMMENTED** [review]
Looks like a good start!

I don't feel strongly about this, but how would you feel about using [myst-parser](https://myst-parser.readthedocs.io/en/latest/) so that we can write docs in Markdown and not need to switch syntaxes between the docs and the README?

**Comment by sethmlarson** [comment]
Loved the idea to use MyST, switched to that in my latest commit.

**Review by davisagli — APPROVED** [review]

**Comment by davisagli** [comment]
The RTD build failed with:
```
/home/docs/checkouts/readthedocs.org/user_builds/truststore/checkouts/57/docs/source/index.md:3: ERROR: Directive 'toctree': Invalid options YAML: mapping values are not allowed here
  in "<unicode string>", line 2, column 18:
    caption: Contents:
                     ^
```

**Comment by sethmlarson** [comment]
Fixed in the latest push, removed the trailing `:` since it doesn't show up anyways.

**Pull request merged**